sdk/src/test/java/com/mapbox/maps/MapControllerTest.kt

```diff
@@ -311,4 +314,32 @@ class MapControllerTest {
       mockMapboxMap.setCameraAnimationPlugin(createdPlugin.instance as CameraAnimationsPlugin)
     }
   }
+
+  @Test
+  fun setValidScreenRefreshRate() {
+    mockkStatic("com.mapbox.maps.MapboxLogger")
+    every { logE(any(), any()) } just Runs
+    val mockRenderThread = mockk<MapboxRenderThread>(relaxed = true)
+    every { mockRenderer.renderThread } returns mockRenderThread
+    val validScreenRefreshRate = 90
+    testMapController.setScreenRefreshRate(validScreenRefreshRate)
+    verify(exactly = 1) {
+      mockRenderThread.setScreenRefreshRate(validScreenRefreshRate)
+    }
+    unmockkStatic("com.mapbox.maps.MapboxLogger")
+  }
+
+  @Test
+  fun setInvalidScreenRefreshRate() {
+    mockkStatic("com.mapbox.maps.MapboxLogger")
+    every { logE(any(), any()) } just Runs
+    val mockRenderThread = mockk<MapboxRenderThread>(relaxed = true)
+    every { mockRenderer.renderThread } returns mockRenderThread
+    testMapController.setScreenRefreshRate(-1)
```

Comment:
Please add a case for  `testMapController.setScreenRefreshRate(0)`

Reply:
ok, will do